CS.API/CS.API.csproj

```diff
@@ -10,7 +10,6 @@
 
   <ItemGroup>
     <PackageReference Include="FluentValidation.AspNetCore" Version="11.3.0" />
-    <PackageReference Include="MailKit" Version="4.4.0" />
```

Comment:
Your API project has reference to BL, and BL imports MailKit, so this one is redundant